index.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Hexal</title>
+    <link rel="stylesheet" href="styles/style.css">
+    <link rel="stylesheet" href="styles/reset.css">
+    <link href="https://fonts.googleapis.com/css?family=Lato|Maven+Pro" rel="stylesheet">
+
+  </head> 
+  <body>
+      <header>
+            <nav class="navigation">
+              <ul class="menu">
+                  <li><a  id='first-item' href="#">PORTFOLIO</a></li>
+                  <li id='separator'></li>
```

Comment:
Don't use empty li for point. Use ::after {...} for creating this point

Reply:
Ok